mkdocs.yml

```diff
@@ -39,6 +39,7 @@ nav:
           - Home: index.md
           - Device Support and State of Features: state.md
           - Installation Roadmap: roadmap.md
+          - Contribute: donations.md
```

Comment:
This is a bit akward, we should decide on either Contribute or donations

Reply:
db0f6f2